lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -2771,12 +2771,13 @@ static Value NearestInterpolate(OpBuilder &b, Location loc,
   return retVal;
 }
 
-static Value BilinearInterpolate(OpBuilder &b,
-                                 Aten__InterpolateSizeListScaleListOp op,
-                                 Location loc, SmallVector<Value> outputSizes,
-                                 Value input, SmallVector<Value> inputSizes,
-                                 SmallVector<Value> scaleValues,
-                                 std::string coordStr) {
+static SmallVector<Value>
+CoordinateTransform(OpBuilder &b, Aten__InterpolateSizeListScaleListOp op,
```

Comment:
nit: I just noticed this, but all these functions should be camel case. Class names start with a capital, but function names do not.